src/components/portal/portal.tsx

```diff
@@ -212,41 +211,21 @@ export class Portal {
         this.parent.appendChild(this.container);
     }
 
-    private removeContainer() {
```

Comment:
I don't think we should remove this function and just rename it to `hideContainer` Those two are doing different things. This one will be called when the element is disconnected from the DOM, and `hideContainer` will be called when visibility changes